kibana/Dockerfile

```diff
@@ -1,55 +1,44 @@
-FROM rhscl/nodejs-6-rhel7
```

Comment:
looks like `rhscl/nodejs-10-rhel7` is now available on registry.redhat.com... i'll try to switch to using that